src/geotiffimage.js

```diff
@@ -616,6 +616,18 @@ class GeoTIFFImage {
     return metadata;
   }
 
+  /**
+   * Returns the GDAL nodata value
+   * @returns {Number} or null
+   */
+  getGDALNoData() {
+    if (!this.fileDirectory.GDAL_NODATA) {
+      return null;
+    }
+    const string = this.fileDirectory.GDAL_NODATA;
+    return Number(string.substring(0, string.length - 1));
```

Comment:
We need to limit the length of the string to exclude the final `\0`, right?

Reply:
> We need to limit the length of the string to exclude the final `\0`, right?

Exactly. Similarly as done for the GDAL_METADATA case

Reply:
Okay, perfect!